### ISSUE TITLE
Date field shown within a group with conditional remains hidden after submit and going back to its page on a multi-pages form

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#503]
**Description**
If a date field is added within a group field on a multi-pages form, and if that date field is shown based on a radio field value within this group, the date field remains hidden after going forward and then going back in this multi-pages form.

**Steps to reproduce**
1. Setup a multi-pages form
2. Add a group field before the last page of the form
3. Within this group, add a radio and a date field
4. Setup a conditional to show the date field based on the radio field value
5. Try a new submission, submit the pages until the next page of the date field
6. Go back to the date field page and check if the date field is showing up

**Additional info**
Plugin version: 1.4.7
Craft version: 3.6.17
Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
